core/math/math.odin

```diff
@@ -48,6 +51,57 @@ clamp :: builtin.clamp
 @(require_results) sqrt_f32be :: proc "contextless" (x: f32be) -> f32be { return #force_inline f32be(sqrt_f32(f32(x))) }
 @(require_results) sqrt_f64le :: proc "contextless" (x: f64le) -> f64le { return #force_inline f64le(sqrt_f64(f64(x))) }
 @(require_results) sqrt_f64be :: proc "contextless" (x: f64be) -> f64be { return #force_inline f64be(sqrt_f64(f64(x))) }
+/*
+Return square root of given input.
+
+**Only accept floats**
+
+Inputs:
+- x: input value of type floats
+
+Output:
+- x: ouput value that with same type of the input
+
+Example:
+        x_float: f64   = 4.0;    sqrt_x_float := math.sqrt(x_float) // using default type of f64
+        x_f16:   f16   = 30.90;  sqrt_x_f16   := math.sqrt(x_f16)
+        x_f16le: f16le = 50.0;   sqrt_x_f16le := math.sqrt(x_f16le)
+        x_f16be: f16be = 100;    sqrt_x_f16be := math.sqrt(x_f16be)
+        x_f32:   f32   = 4.89;   sqrt_x_f32   := math.sqrt(x_f32)
+        x_f32le: f32le = 3.14;   sqrt_x_f32le := math.sqrt(x_f32le)
+        x_f32be: f32be = 2.0;    sqrt_x_f32be := math.sqrt(x_f32be)
+        x_f64le: f64le = 1000.0; sqrt_x_f64le := math.sqrt(x_f64le)
+        x_f64be: f64be = 89.98;  sqrt_x_f64be := math.sqrt(x_f64be)
+
+        // special cases, (see Float_Class and math.classfiy)
+        y_f64_pos_zero: f64 = +0.0;             sqrt_y_f64_pos_zero := math.sqrt(y_f64_pos_zero) // +0.0
+        y_f32_neg_zero: f32 = -0.0;             sqrt_y_f32_neg_zero := math.sqrt(y_f32_neg_zero) // -0.0
+        y_f16_pos_inf:  f16 = math.inf_f16(+1); sqrt_y_f16_pos_inf  := math.sqrt(y_f16_pos_inf)  // +Inf
+        y_f32_zero_inf: f32 = math.inf_f32(0);  sqrt_y_f32_zero_inf := math.sqrt(y_f32_zero_inf) // Inf
+        y_f64_neg_inf:  f64 = math.inf_f64(-1); sqrt_y_f64_neg_inf  := math.sqrt(y_f64_neg_inf)  // -Inf
+        y_f64be_nan:  f64be = math.nan_f64be(); sqrt_y_f64be_nan    := math.sqrt(y_f64be_nan)    // NaN
+        y_f16le_nan:  f16le = math.nan_f16le(); sqrt_y_f16le_nan    := math.sqrt(y_f16le_nan)    // NaN
+
+Output:
```

Comment:
Your example doesn't "print" anything, so there's no "output". This is going to be a problem with the documentation tester that runs all examples inside documentation and checks the output. I suggest you reformat these examples to print results. Might help to not focus on types as much as on values.